### ISSUE TITLE
The long tail of contact tracing

### ISSUE THREAD

**Issue by InstitutefTiPI** [DP-3T/documents#118]
We raise this issue out of concern for the way "contact tracing apps" are being proposed without taking into account the complexity of the social. The current discussions present only two options for living: one of surveillance and the other of confinement. We demand the possibility to imagine something else for future cohabiting.

The current rush towards contact tracing apps is at risk of framing COVID-19 exposure as an exclusively technical problem, therefore privileging technological expertise as the single site of improvement. The number of issues being opened in this thread that are of a technical nature, confirms that this is happening. We are calling for an expanded, careful discussion on the design and implementation of contact tracing apps. We need to take into account the complex differing social relations, and this discussion needs to involve various voices, not just those of governments, engineers, epidemiologists and other authorities.

We think it is necessary and overdue to rethink the way technology gets designed and implemented, because contact tracing apps, if implemented, will be scripting the way we will live our lives and not just for a short period. They will be laying out normative conditions for reality, and will contribute to the decisions of who gets to have freedom of choice and freedom to decide ... or not. Contact tracing apps will co-define who gets to live and have a life, and the possibilities for perceiving the world itself. This means to consider the way technologists and technologies are infused with social norms in and of themselves; however the discussion needs to consider technologists as having something at stake as well. 

We also need to remember that these are not new concerns. The history of critical debate and radical interventions in technological hegemony offers a rich archive of how technological solutionism without resistance and awareness, inevitably will be complacent with racialised capitalism which amounts to unequal intersections of race, class, ability and gender.

**Epidemiology, surveillance and public health**

- Epidemiology and public health have served racial, class, sexual, ableist divides and colonial practices. For example in the "Tuskegee Study of Untreated Syphilis" a clinical study where the subjects of the study were only told they were receiving free health care from the Federal government of the United States. https://en.wikipedia.org/wiki/Tuskegee_syphilis_experiment 
- In addition, we urgently need to look at the violence that is produced when surveillance and epidemiology are working together. See for example the brutalities mobilised in the name of surveillance during the AIDS epidemic which clearly shows the issues with contact with health authorities and commercial partners.

**Home or surveillance?**

- We are currently being presented only two options, a paradigm of either implementing a contact tracing app or continuing to social distance at "home".  However this appears to be a false dichotomy. Is tracing contacts the preferred response? What are other options? 
- Why has surveillance become the only device through which we can effectively act? Why has surveillance become the only possibility for action?
- We need to be able to discuss publicly how communities might manage their own care and exposure to this virus and possibly other ones. This needs to include whether if it is an app, if it should be a decentralised or centralised architecture but also the possibility of NOT deploying the technology.

**Who decides?**

- Will communities be given the decision making power? We need to ask for what and for whom is this exposure tracing app, and how does it fulfill or not the ways in which communities want to protect themselves and manage exposure. The urgent challenge is how to facilitate a public discussion so that communities can design, test and shape the methods for protection, care and managing their own exposure.
-  How can this be done without replicating a reduced version of participatory design that involves predefined representatives of the state, business, academia and sometimes communities that are identified as 'having a concern'. We instead propose to call for accomplices that can together share in the risks because they have skin in the game. One only has to think of existing racial profiling that takes place in similar apps, producing and reproducing racial capitalism, or the explicitly able-bodied bias in most design approaches, to see why this is necessary.
-  We cannot assume that all communities want and need the same approach to managing exposure via mobile devices, nor that they want or need the same universal solutions. How can we critically discuss and continuously assess who this technology is designed to protect, and who is being externalised?
-  This proposal for Decentralized Privacy-Preserving Proximity Tracing is important because it includes a much needed reflection on how and when the app would get gracefully dismantled once users stop using it. But there might be external reasons for using the technology, such as getting access to a public venue or even a residential building. It's decentralized architecture seems more reasonable than proposing centralized control but by distributing responsibility to individuals, we need to take into account that to stop using the app might not be an individual choice to make.
-  What can we learn from those playing with 'old' technologies and infrastructures and manipulating them in a way that does not service racial capitalism? Perhaps rigging is what we need, more than designing. What does it mean that technologists prefer be inventors instead of re-users. This colonial discourse is the very mindset that also fueled the first-world-saviour-complex during the Arab revolutions.
- Infrastructures need to include human AND technological creativity. Its is important to remember that there are still things we can do without technology. For example there is still a broader political value in direct human-to-human communication for rebellious potential. The risk of forgetting this, is a turn to deconstructionism that also risks demobilising human collaboration.

**Not-being-infected as the norm**

-  In what way can we avoid assuming that not-being-infected is the norm? How do we avoid the (re)establishing of a condition that a good citizen or labourer should prove their, her or his healthiness? How can we deal with the responsibility placed on the individual to prove health? How do we avoid establishing the universal assumption that all people wish not to be exposed (or infected)? Do these apps automatically privilege those that are not infected, for example assuming that they do not need to be subjected to surveillance?
-  More generally, how can technology be challenged to consider its own norms, stakes and risks more critically. Engineers are part of society and being part of the conversation, but not the only part of it and not the only ones with expertise. A thinking through the relation between tech and the social needs to be anchored in any tech implementation.
-  We need to recognize engineers as part of the complex social, with their own norms and blindspots.

**Extractive infrastructures**

- Contact tracing apps will increase the reliance on mobile devices, which in turn will necessitate an increased extraction of resources and increased levels of toxic exposure for involved communities, resulting from extraction. We have to ask who bears the body burden of contact tracing apps, who will experience more bodily harms from their use? The propositions of the app as the preferred answer to the crisis, is a continuation to pursue the expansion of computational infrastructures, for the protection of some lives at the cost of others, resulting in producing and reproducing racial capitalism.
- Smart phones and computing devices are the biggest industrial users of gold. The extraction of gold creates a variety of occupational and environmental hazards related to and including mercury exposure, deforestation and contamination of water that has significant effects on communities living with extraction. In this COVID-19 response, we need to ask what is the calculation being made here, what values are being placed on lives and exposures, who's lives come to matter or not? What does it mean to develop technologies to combat a virus without considering the larger impact and toxic exposures? How can the app mitigate these harms? Is that even possible? 
- How to facilitate a public discussion so that communities can design, test and shape methods for protection not from viruses, but from extractive modes of existence.
- Contact tracing apps will rely on the increased use and new adoption of cloud infrastructures and commercial communication networks which are maintained and monetized by multinationals. They extend the public health market further for these companies and provides a moment in which governments, health bodies and communities have a renewed dependence on them. How can we ensure that multinationals through the implementation of contact tracing app services, do not capitalize on life and death for economic gains? 

Decentralized Privacy-Preserving Proximity Tracing has generously provided a space for discussion (which almost all other contact tracing or health projects that are proposed or deployed do not) and we appreciate this. The issues we raise may seem to overburden this project but they will need to be addressed nevertheless. The moment of COVID-19 should not just be about business as usual, with all the same power relations and harms in place but instead can be a moment to together transform our practices.

On behalf of Institute for Technology in the Public Interest,

Miriyam Aouragh
Helen Pritchard
Femke Snelting

**Comment by kennypaterson** [comment]
Thank you for this thought-provoking contribution. Our core project participants collectively have some awareness of the issues your post raises and we have had some internal debate on a limited number of the aspects. However, I don't think any one of us could express this set of concerns in such a clear and comprehensive manner as you have. I hope we can use this forum to start to address them, with your help. 

I would also remark at this point that these points apply to all the parallel technology-based contact tracing initiatives that are underway around the world (there are dozens, including several already in operation in, for example, Singapore and Austria). At the same time, as you remark, most of those projects do not provide a forum where these concerns can be debated, so here we are.

So: to begin to dig into the questions you raise: you report that we are currently being presented with two options, a paradigm of either implementing a contact tracing app or continuing to social distance at "home", and that this appears to be a false dichotomy. My (limited) understanding of the epidemiology indicates that contact-tracing is effective in limiting the spread of COVID-19, but that it is labour-intensive and error-prone when done manually; hence a growing focus on using technology to assist in the process.

You suggest there could be other options. Concretely, could you share your ideas for what these might be? I apologise if this seems overly focussed on problem solving rather than promoting a broader exploration of the ideas you have expressed; however, we need to start somewhere, and I can't think of a better question to focus on than the very first one you raise.

**Comment by veale** [comment]
I also want to emphasise that GitHub Issues as a tool is built in a very linear way (e.g. to 'close' issues) — and this is clearly a broader question and set of issues like that. So I suggest for noe we use this thread to offer more discussion as an initial, small forum, thinking in particular (as it would be extremely useful) around what the DP-3T project can do to support the furthering of some of these questions in the current context, and in light of the context in which this project operates.

**Comment by tzaeru** [comment]
Thanks for the thought-provoking write-up, there's good and interesting points raised there. I think it further underlines the difficulties in creating e.g. global or even EU-wide solution and also adds fuel to the idea that an applications like this, done in a very short period of time, should be strongly distributed and anonymous rather than centralized; as to limit their usability for surveillance.

One sociological aspect about this coronavirus crisis that I've thought of a lot has been what kind of a precedent does it set. In the extremes, will the precedent be that when shit hits the fan, strong states are needed to utilize emergency laws, mass surveillance and other potentially dystopian measures or will the precedent be that when shit hits the fan, people are able to come together naturally, fix things together and trust each other to e.g. use non-centrally authorized applications honestly and correctly?

These applications will be setting a precedent, too. Personally I hope that the precedent ends up being that we can use distributed, anonymous technology to solve problems responsibly rather than that it's not possible to do this without a strong central authority for the data.

**Comment by jjtmp** [comment]
An example for a concept where people should/"want" get infected can be found here: http://jbechtel.de/site/sonstiges/covid19/CDCC/

Unfortunately South Korea has found that after being infected, you aren't necessarily immune (about 91 persons got sick again). And few people actually want to move two times, I guess.

But maybe some adventurous community will do this.

**Comment by burdges** [comment]
Just fyi, TCN merges a PR that includes narrower comments: https://github.com/TCNCoalition/TCN/pull/58

All contact tracing proposals assume far more than merely that "not-being-infected [is] the norm".  

All contact tracing proposals work under the assumption that only some _negligible_ fraction gets infected, but that infected people must sacrifice their freedom of association and medical privacy.  All publicly discussed privacy preserving contact tracing designs aim merely to preserve freedom of association for uninfected people.  Just fyi, there are zero-knowledge proof tricks that improve privacy for infected people somewhat, like say each participant can compromise the infection status for "one other participant per day", but that's another topic and not so amazing. 

Is this assumption that only a negligible fraction gets infected reasonable?  In a sense yes..  It's a prior assumption on contact tracing itself:

If infection rates are negligible, then contact tracing helps health services react faster, which limits an otherwise exponential spread.  In Switzerland, there is one non-government organization talking about contact tracing once the entire country drops below 25 new cases per day. It's possible they are being conservative, but a denser area like NYC require even smaller numbers.

If however infection rates are non-negligable, then contact tracing cannot help control an epidemic anyways because reaction limited by other factors including individual behavior.  In fact, _contact tracing could cause infections and deaths_ because it provides an flawed excuse to restart economic activity.

There are never public discussions about thresholds for threshold in the media in the U.S. and U.K., who instead pushed contact tracing into their media pump cycle.  These anglophone countries might easily make contact tracing into an excuse to ignore necessary economic restrictions, which sadly fits their earlier intention to ignore the epidemic entirely.

In short, an epidemiological model used to justify contact tracing should be public so that people can know if they're risking medical privacy to save lives, or because they value the economy more than lives.

**Comment by jaromil** [comment]
Thanks for opening this issue and for its quality.

FWIW I've shared my opinion and experience in this post  https://medium.com/@jaromil/why-proximity-tracing-is-important-and-its-integrity-should-be-contextual-2b46e5681a45

**Comment by veale** [comment]
Hi all,
Speaking not as part of DP-3T, but as a small group from UK legal academia and civil society, we have put together a draft Coronavirus (Safeguards) Bill to add protections against the use and conditionality of these technologies, as well as other forms of legal oversight. It's available to download here: https://osf.io/preprints/lawarxiv/yc6xu

**Comment by Jay-Lewis** [comment]
All, 

First, I wanted to say that I appreciate the gitub repo and the work that has been done to provide an approach which affords an appreciable level of privacy. As an initial framework, I think it allows for a way to begin alleviating the outbreak we are facing. That being said, I sympathize with the response made on behalf of  the Institute for Technology in the Public Interest. As they mentioned, I believe that developing a contact tracing system should be done in a way which is forward thinking (assuming it should be developed at all). Contact tracing runs a great social risk which will certainly outlast the current storm the world is facing. 

Along these lines, I wanted to bring to your attention work I've been a part of recently which is aiming at implementing contact tracing while preserving the privacy of users in a way which is fundamentally different from currently purposed technologies. The system, **SafeTrace**, is decentralized and based on multi-party computation (MPC) algorithms (see the whitepaper on our website: https://safetraceapi.org/). By leveraging MPC algorithms, a contact tracing system is possible without the need for a trusted, centralized authority.

Please note that with our approach, user data is **not** given in its direct form to anyone for computational purposes. Instead, each piece of data is broken into cryptographic shares which are given to a set of decentralized servers (each server is owned by a distinct entity). Then, the computation is done on the shares. Given the assumption that the computing parties / servers do not collude, this approach affords similar privacy guarantees afforded by symmetric key, homomorphic encryption.

Note that this privacy guarantee does not come for free. There is a substantial computational overhead which arises due to the MPC algorithms. Despite these challenges, we are nearing the completion of a prototype which demonstrates that developing a contact tracing system in this way is scalable.

 Our team would love to get in contact with the authors of this github page as well as those that have commented (my email: justin94lewis 'at' utexas 'dot' edu). SafeTrace is a global collaboration which has gathered together epidemiologists, computer scientists, and economists from across numerous academic institutions, corporations and governmental agencies. That being said, we'd love for anyone to join our cause / work with us. All the best.

**Comment by jaromil** [comment]
Many thanks for this bill @veale, it is very valuable also considering the EU is signing a first draft of the strategy tomorrow I believe this is all applicable and should be also to frameworks outside the UK.

**Comment by kholtman** [comment]
@InstitutefTiPI  Just found this discussion; I agree with your assessment and that of others above that the complexity the social forces and needs surrounding the technology is currently underestimated.  For similar conclusions and points of view, please see #224 on the Dutch experience and #239 where I proposing to use 'harm stories' as an analytical tool to move to a higher level of understanding of what we really mean when our gut feeling tells us that 'decentralised' is the preferred approach.

**Comment by pdehaye** [comment]
I would like to point folks here to the issue [here](https://github.com/digitalepidemiologylab/swisscovid_efficacy/issues/4), and also corresponding issues at that repository. 

In short: there is real reason to not only expect that digital contact tracing would be discriminatory in its impact, but also that the most obvious metrics that could be used would precisely hide that fact. 

**Disclaimer: This is my personal opinion, and does not necessarily reflect the opinion of any of my employers or clients.**
